src/site/tests/home/00-required.e2e.spec.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+const path = require('path');
+
+const E2eHelpers = require('platform/testing/e2e/helpers');
+
+function verifyGoogleAnalytics(browser) {
+  const googleAnalyticsCode = fs.readFileSync(
+    path.join(
+      __dirname,
+      '../../assets/js/google-analytics/',
+      `${process.env.BUILDTYPE}.js`,
+    ),
+  );
+
+  browser.execute(
+    () => document.querySelector('[data-e2e="analytics-script"]').innerHTML,
+    [],
+    ({ value: analyticsScriptHtml }) => {
+      const codeExists = analyticsScriptHtml.includes(googleAnalyticsCode);
+      browser.assert.equal(
+        codeExists,
+        true,
+        'Google Analytics code exists in the DOM.',
+      );
+    },
+  );
+}
+
+function verifyElementCount(browser, selector, expectedNumElements, message) {
+  browser.execute(
+    data => Array.from(document.querySelectorAll(data)).length,
+    [selector],
+    ({ value: numElements }) => {
+      browser.assert.equal(numElements, expectedNumElements, message);
+    },
+  );
+}
+
+function main(browser) {
+  browser.openUrl(`${E2eHelpers.baseUrl}/`).axeCheck();
+
+  verifyElementCount(browser, '[data-e2e="bucket"]', 4, 'There are 4 buckets');
+  verifyElementCount(browser, '[data-e2e="hub"]', 11, 'There are 11 hubs');
+  verifyElementCount(
+    browser,
+    '[data-e2e="news"]',
+    3,
+    'There are 3 news stories',
+  );
```

Comment:
I like this because this E2E test might catch issues with the homepage template. It's possible bad content could be consumed during a content deploy and bypass the E2E test, but I think it's still better than nothing.